bundle/config/validate/files_to_sync_test.go

```diff
@@ -74,6 +73,10 @@ func setupBundleForFilesToSyncTest(t *testing.T) *bundle.Bundle {
 		ObjectType: workspace.ObjectTypeDirectory,
 	}, nil)
 
+	m.GetMockCurrentUserAPI().EXPECT().Me(mock.Anything).Return(&iam.User{
```

Comment:
Why is this call now needed and wasn't needed before?

Reply:
Great question. The  minimal user is indeed set up by a test, so it should not be needed. The reason it was needed was that mutator was called via bundle.Apply() which wiped out user in MarkMutatorEntry() code.

Why did wipe out occur I did not investigate.

Switching to calling mutator directly no longer does that. https://github.com/databricks/cli/pull/2414/commits/446c1c1ff40ca69006203c8799d7e3dc053776e5